sys/socket.c

```diff
@@ -555,3 +564,16 @@ void freeifaddrs(struct ifaddrs *ifa)
 {
 	free(ifa);
 }
+
+const char *hstrerror(int err)
+{
+	switch (err) {
+		case RESOLVED_OK: return "SUCCESS";
```

Comment:
Please, add return in a new line.

Reply:
ok, `clang-format` allowed this construction, resolving